vm/adb/adb.go

```diff
@@ -148,6 +148,8 @@ func (pool *Pool) Create(workdir string, index int) (vmimpl.Instance, error) {
 		return nil, err
 	}
 	if inst.console == "" {
+		// More verbose log level is required to get sn shown in /dev/kmsg.
+		inst.adb("shell", "echo 8 > /proc/sys/kernel/printk")
```

Comment:
Does not begin to print too much unrelated information?

Reply:
yes, init.rc will override it to loglevel 4. so the findConsoleImpl will failed to get the sn.